tests/baselines/reference/user/jimp.log

```diff
@@ -0,0 +1,9 @@
+Exit Code: 1
+Standard output:
+node_modules/@jimp/core/types/utils.d.ts(16,3): error TS2536: Type '"class"' cannot be used to index type 'T'.
```

Comment:
```ts
export type WellFormedValues<T extends any> =
  T['class'] &
  T['constants'];
```

This change is intended, but I'm not sure how to fix this break. @weswigham is there an easy workaround?

Reply:
Here's *a* fix:

```ts
export type WellFormedValues<T extends { class: any, constants: any }> =
    T['class'] &
    T['constants'];
export type GetPluginVal<Q> = Q extends Required<{class: any, constants?: any }> | Required<{constants: any, class?: any }>
  ? WellFormedValues<Q>
  : Q;
```

I added `constants?: any` and `class?: any` to the second type to make it compile, which might cause downstream failures.